demos/projects/ST/b-l475e-iot01a/CMakeLists.txt

```diff
@@ -60,3 +60,42 @@ add_custom_command(TARGET ${PROJECT_NAME}
     COMMAND ${CMAKE_OBJCOPY} -Obinary $<TARGET_FILE:${PROJECT_NAME}> ${PROJECT_NAME}.bin
     COMMENT "Generate Bin file"
     VERBATIM)
+
+# Add PnP Sample
+add_executable(${PROJECT_NAME}-pnp ${PROJECT_SOURCES})
+target_include_directories(${PROJECT_NAME}-pnp PUBLIC
```

Comment:
currently in gcc_flags we set the map file to be output.map, and now we have two target generated. This will cause the later target to override previous map file.